resources/scripts/components/server/backups/CreateBackupButton.tsx

```diff
@@ -80,9 +80,9 @@ export default () => {
     const submit = (values: Values, { setSubmitting }: FormikHelpers<Values>) => {
         clearFlashes('backups:create');
         createServerBackup(uuid, values)
-            .then((backup) => {
-                mutate(
-                    (data) => ({ ...data, items: data.items.concat(backup), backupCount: data.backupCount + 1 }),
+            .then(async (backup) => {
```

Comment:
Could this async lead to a race condition given this is a network operation? Not sure why this specifically pokes out to me.

Reply:
It shouldn't be a big deal since the req that creates a backup is only a couple bytes at its most. There should be no reason this fails and causes any major issues imo

Reply:
Ahh, I thought it would be the size of the actual backup, all is well then ^^